qiskit/providers/aer/noise/noise_model.py

```diff
@@ -302,11 +302,15 @@ def from_backend(cls, backend,
         Raises:
             NoiseError: If the input backend is not valid.
         """
-        if isinstance(backend, (BaseBackend, Backend)):
+        if isinstance(backend, BackendV2):
+            raise NoiseError(
+                "V2 Backends are not yet supported by `NoiseModel.from_backend`")
+        if isinstance(backend, (BaseBackend, BackendV1)):
             properties = backend.properties()
-            basis_gates = backend.configuration().basis_gates
-            num_qubits = backend.configuration().num_qubits
-            dt = backend.configuration().dt
+            configuration = backend.configuration()
+            basis_gates = configuration.basis_gates
+            num_qubits = configuration.num_qubits
+            dt = getattr(configuration, "dt", 0)
```

Comment:
I'd be tempted to restore this to the previous behaviour, especially this close to release.  Now that we're filtering for `BackendV1`, those properties should be guaranteed to exist in those locations.